docs/rules/no-fallthrough.md

```diff
@@ -54,6 +54,17 @@ switch(foo) {
     case 2:
         doSomethingElse();
 }
+
+switch(foo) {
+    case 1: {
+        doSomething();
+        // falls through
+    }
+
+    case 2: {
+        doSomethingElse();
+    }
+}
```

Comment:
Can you also add this same example to the "Examples of correct code for this rule" code block?

Reply:
Done.